### ISSUE TITLE
`skip cd` tag inefficiencies

### ISSUE THREAD

**Issue by ROpdebee** [ROpdebee/mb-userscripts#600]
The `skip cd` tag on PRs can be used to prevent deploying updates when it's undesired to push an update. For example, #593 and #598 cause changes to userscripts which are not worth deploying an update for.

However, merging these with `skip cd` wouldn't prevent the meaningful updates entirely, it would only prevent them from being pushed immediately when the PR is merged. If another change happens later on that doesn't involve changing the scripts (e.g. updating a dev dependency), the changes would still be deployed. In those cases, the changelogs would also be wrong. Instead, the updates should be postponed until the next change that actually changes the userscripts.

In other words: Deployment automation currently fires when 1) there is no `skip cd` tag and 2) the built scripts are different from the deployed scripts.
Instead, it should be changed to 1) there is no `skip cd` tag and 2) the built scripts are different from the last version before the change (even if that last version wasn't deployed).
